app/src/main/java/org/p2p/wallet/history/ui/token/adapter/HistoryAdapter.kt

```diff
@@ -114,6 +128,8 @@ class HistoryAdapter(
                     oldItem.transactionId == newItem.transactionId && oldItem.statusIcon == newItem.statusIcon
                 oldItem is DateItem && newItem is DateItem ->
                     oldItem.date.isSameAs(newItem.date)
+                oldItem is UserSendLinksItem && newItem is UserSendLinksItem ->
+                    oldItem.linksCount == newItem.linksCount
```

Comment:
what about `areItemsTheSame` correction?

Reply:
Do I really need to refactor the history nooow? 💀

Reply:
not the history, but at least modify `areItemsTheSame ` i guess. Or is it working correct now?

Reply:
nothing extraordinary, I guess. let's see will there be problems from qa?